ingest.py

```
@@ -11,53 +11,92 @@
 
 try:
     import requests
 except ImportError:
-    print("Installing requests...")
+    print("Warning: requests package not found, attempting to install...")
     import subprocess
-    subprocess.check_call([sys.executable, "-m", "pip", "install", "requests", "--break-system-packages", "-q"])
-    import requests
+
+    try:
+        subprocess.check_call(
```

Ruff (0.14.8)

`subprocess` call: check for execution of untrusted input

(S603)